pkgs/development/python-modules/sphinxcontrib-wavedrom/default.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, setuptools-scm
+, sphinx
+, wavedrom
+, xcffib
+, cairosvg
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-wavedrom";
+  version = "3.0.4";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-0zTHVBr9kXwMEo4VRTFsxdX2HI31DxdHfLUHCQmw1Ko=";
+  };
+
+  nativeBuildInputs = [ setuptools-scm ];
+
+  propagatedBuildInputs = [
+    sphinx
+    wavedrom
+    xcffib
+    cairosvg
+  ];
+
+  # No tests included
+  doCheck = false;
+
+  pythonImportsCheck = [ "sphinxcontrib.wavedrom" ];
+
```

```suggestion

  pythonNamespaces = [ "sphinxcontrib" ];

```
Could you please add `pythonNamespaces`?
Without it, the sphinx version upgrade may break this package.
see more details, https://github.com/NixOS/nixpkgs/pull/259350